terraform/modules/common/ami/main.tf

```diff
@@ -0,0 +1,33 @@
+## Variables
+
+variable "amazon_release" {
```

Comment:
Is this expected to always be a version? Would appending `_version` on the end help communicate this if so? Or maybe `ami_release_version`?

Reply:
Think so, though  I'm not sure if there needs to be any change as it's specific to the amazon release